CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+### 0.0.7
+
+FEATURES:
+
+* [29](https://github.com/nginxinc/nginx-ingress-operator/pull/29) Add IngressClass support.
+* [26](https://github.com/nginxinc/nginx-ingress-operator/pull/26) Add mTLS policy support.
+* [25](https://github.com/nginxinc/nginx-ingress-operator/pull/25) Add JWT policy support.
+* [21](https://github.com/nginxinc/nginx-ingress-operator/pull/21) Add latency metrics support.
```

Comment:
should we include this even tho theres a bug with it?

Reply:
Added a note @LorcanMcVeigh